certora/specs/CER-2-Operator/CER-54-Operator-owner.spec

```diff
@@ -0,0 +1,23 @@
+import "../utils/IsMustRevertFunction.spec";
+import "CER-68-Operator-authorization.spec";
+
+methods {
+    function getOwnerOf(bytes19) external returns (address) envfree;
+    function getOperator(bytes19, address) external returns (uint256) envfree;
+    function getAddressPrefix(address) external returns (bytes19) envfree;
+    function haveCommonOwner(address account, address otherAccount) external returns (bool) envfree;
+}
+
+// CER-54: Account Operator address MUST NOT belong to the Account Owner of the 
```

Comment:
this must also be proved for `setOperator`

Reply:
This is implemented now. See what you think of it.